torch/csrc/jit/codegen/cuda/transform_replay.h

```diff
@@ -116,11 +116,10 @@ namespace fuser {
  *
  */
 
-struct TensorDomain;
-struct TensorView;
+class TensorDomain;
+class TensorView;
 
-struct TORCH_CUDA_API TransformReplay {
- private:
```

Comment:
Did you mean to remove this `private:` label?

Reply:
There wasn't anything in it, was fine to remove.